pymodbus/transport/transport.py

```diff
@@ -58,6 +58,8 @@
 from functools import partial
 from typing import Any
 
+from serial.rs485 import RS485Settings
```

Comment:
What happens if serial is not installed ?? Please import as serial is imported in other places.